apps/web/src/common/components/JobCard/JobCardFooter/UnpaidFooter/index.tsx

```diff
@@ -0,0 +1,22 @@
+import Button from '@mui/material/Button/Button'
```

Comment:
It's not directly tree-shakeable but for global styling
```suggestion
import { Button } from '@mui/material'
```